time-split/control.lua

```diff
@@ -1 +1,179 @@
--- control
+-- TODO: this is not compatible in multiplayer mode
+local player_index = 1
+local timestamp_pattern = "(%d%d):(%d%d):(%d%d)"
+local DEFAULT_TICK = 2592000
+
+function padded_to_string(number)
+    if number < 10 then
+        return "0" .. tostring(number)
+    end
+    return tostring(number)
+end
+
+function space_padded(str)
+    return "  " .. str .. "  "
+end
+
+function tick_to_timestamp(tick)
+    local seconds = tick / 60
+    local hours = math.floor(seconds / 3600)
+    seconds = seconds % (3600)
+    local mins = math.floor(seconds / (60))
+    seconds = math.floor(seconds % (60))
+    return padded_to_string(hours) .. ":" .. padded_to_string(mins) .. ":" .. padded_to_string(seconds)
+end
```

Comment:
Consider only including units that has a value greater than 0, i.e. 00:00:12 -> 12 and 00:32:23 -> 32:23 (truncate=false?)